identity-service/src/relay/txRelay.js

```diff
@@ -66,7 +66,8 @@ const sendTransaction = async (
   req,
   resetNonce = false,
   txProps,
-  reqBodySHA
+  reqBodySHA,
+  reqStartTime,
```

Comment:
this variable isn't read but i think this should just be created in sendTransactionInternal so it doesn't have to start at each each

Reply:
oh good catch, it was meant to be passed through

Reply:
we want to track the whole routes timing though right? if we start it in `Internal` it wouldn't be correct

Reply:
i was really only concerned with /relay and forgot that the social media ones will use it too. i think as it is now, it could actually be deceptive like what if something in tiktok takes longer than the normal /relay. so i think just starting the latency in this common place would make more sense. what do you think?